user_modules/connecttest_iap2_emulation.lua

```diff
@@ -0,0 +1,109 @@
+local module = require("user_modules/dummy_connecttest")
```

Comment:
@dev-gh Propose to redesign scripts according new approach, described in [template_of_script](https://github.com/smartdevicelink/sdl_atf_test_scripts/blob/develop/test_scripts/template_of_script.lua)